### PR TITLE
src/initialize: add zarhus private repos from gitlab

### PR DESCRIPTION
**Pull request by pietrushnic**

### CODE REVIEW

File: contribution-stats [hunk 1]
```diff
@@ -597,65 +597,120 @@ initialize() {
...
+  # 	["meta-trenchboot"]="https://github.com/3mdeb/meta-trenchboot.git"
+  # 	["meta-trenchboot-attestation"]="https://github.com/3mdeb/meta-trenchboot-attestation.git"
+  # 	["meta-trenchboot-demo"]="https://github.com/3mdeb/meta-trenchboot-demo.git"
+  # 	["meta-vbox"]="https://github.com/3mdeb/meta-vbox.git"
+  # 	["meta-virtualization"]="https://github.com/3mdeb/meta-virtualization.git"
+  # 	["meta-zulu-java"]="https://github.com/3mdeb/meta-zulu-java.git"
+  # 	["oe-meta-go"]="https://github.com/3mdeb/oe-meta-go.git"
+  # )
+
+  # Other 3mdeb contribution
```
Comment (macpijan) [inline contribution-stats:680]: I am not sure if I get the idea here. Do we want these to be commented out?

Comment (pietrushnic) [reply contribution-stats:680]: @macpijan I didn't have time to improve it. I just needed coreboot and Zarhus-related contributions without syncing all heavy repositories, so I created those lists. There is a better way to achieve the same result. We should have dedicated lists for areas of contribution: Zarhus, Dasharo, and PET. 

One additional thing I miss and have in the private repo is an analysis of customers' code to realize how much we developed for them compared to how much we upstreamed. Statistics are sad. In the long run, we would like to prove that we use our product code in customer-dedicated repositories and, of course, upstream more of that, which would mean the customer-specific code base would decrease.

To sum up, there are multiple goals here:
- measure contribution and upstreaming in Dasharo-related repositories
- measure code developed for customers in the scope of Dasharo and Zarhus
- measure public code vs. upstreamed Zarhus-related code 
- measure public code vs. upstreamed Dasharo-related code 
- etc.

Comment (pietrushnic) [reply contribution-stats:680]: @macpijan ping

Comment (macpijan) [reply contribution-stats:680]: So how it works now? I uncomment the repos I want, then run the tool? Do you want to merge this?
Would adding extra parameter to sync/list work, to select group of repositories?

Comment (pietrushnic) [reply contribution-stats:680]: Yes, I'm ok with that. Moving repos to some external file is even better (yaml in `repos` directory?). This is another story where we want to manage the ability to use private and public repositories.

We want to query any set of Dasharo and Zarhus repositories. So far, I use this script once a quarter, but I imagine it twice when we have the Zarhus User Group or similar meetings for customers. We already know some of them would like to use a commercial version of Zarhus and align their development with the Zarhus roadmap. Stats would be very helpful in building credibility as well as marketing and sales stories. That is about value, but speaking about interface I'm not sure about it. 

The current interface has limitations. As you pointed out, modifying the code and regenerating things is not exactly user-friendly. I will propose those YAML files, which will keep repos and then initialize.sh would load those according to the provided parameter (zarhus, dasharo, zarhus-private, dasharo-private, etc.)

Comment (macpijan) [reply contribution-stats:680]: I use it at least once per month, and I'm interested in all 3mdeb contributions. Which mostly are around coreboot anyways. But not always.

Comment (pietrushnic) [reply contribution-stats]: @macpijan please check support for repo-def which I pushed here: https://github.com/3mdeb/contribution-stats/pull/9/commits/b3b98246c663ea89d6c9dc486d8e06a0d3881e46

Comment (macpijan) [reply contribution-stats]: lgtm
